.gitignore

```diff
@@ -183,11 +192,48 @@ $RECYCLE.BIN/
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
-.idea/
+# User-specific stuff
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
+
+# Generated files
+.idea/**/contentModel.xml
+
+# Sensitive or high-churn files
+.idea/**/dataSources/
+.idea/**/dataSources.ids
+.idea/**/dataSources.local.xml
+.idea/**/sqlDataSources.xml
+.idea/**/dynamic.xml
+.idea/**/uiDesigner.xml
+.idea/**/dbnavigator.xml
+
+# Gradle
+.idea/**/gradle.xml
+.idea/**/libraries
```

Comment:
This feels like a step backwards to me. We should continue to ignore all of `.idea/` instead of this laundry list of files.

Reply:
I guess it comes from Cookiecutter, we should fix first there :)

Reply:
If cruft prevents us from improving things locally as compared to the cookiecutter baseline, that feels like a non-starter to me.

Reply:
Agreed. This also needs to be addressed at the template level.

Reply:
And to your comment Glenn, we can always ignore individual files that we do not want to keep in sync in the `pyproject.toml` file. I do agree that this is rather heavy-handed but since currently we do not have any automatic way of keeping downstream files in sync I believe that this should still be an improvement over the current state, what do you think?

Reply:
> If cruft prevents us from improving things locally as compared to the cookiecutter baseline, that feels like a non-starter to me.

My overall approach would be, `cruft` helps to adhere to the cookiecutter. I we do an improvement here (we deviate), `cruft` should raise a warning, and we should focus on change the cookiecutter reference to include the improvement.
I mean, when improving, deviations are expected, and this should help to keep in the right track (not enforcing)

Reply:
That is an interesting idea, we will need to hash out the corresponding process further

Reply:
To give a quick update here, there is an issue that mentioned the ability to skip specific patches, it is closed however because another part of the issue was implemented. I do believe that this feature specifically wasn't implemented, but I have gone ahead and asked that [here](https://github.com/cruft/cruft/issues/58).